guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -74,5 +74,5 @@ If the file exists, run the second `{foreman-installer}` command that updates ce
 +
 IMPORTANT: Do not delete the certificate archive file after you deploy the certificate.
 It is required, for example, when upgrading {ProjectServer}.
-. On a computer with network access to {ProjectServer}, navigate to the following URL: `\https://{foreman-example-com}`.
+. On a computer with network access to {ProjectServer}, navigate to the following URL: `https://{foreman-example-com}`.
```

Comment:
Would something like this solve our issue?

```suggestion
. On a computer with network access to {ProjectServer}, navigate to the following URL:
ifdef::satellite[]
`https://{foreman-example-com}`.
endif::[]
ifndef::satellite[]
`\https://{foreman-example-com}`.
endif::[]
```

Reply:
Likely, but if you go that route I'd probably use an attribute.